app/src/main/java/org/fossasia/openevent/app/data/session/Session.java

```diff
@@ -75,4 +75,8 @@
     @JsonSerialize(using = ObservableStringSerializer.class)
     @JsonDeserialize(using = ObservableStringDeserializer.class)
     public ObservableString endsAt = new ObservableString();
+
+    @ColumnIgnore
+    @JsonIgnore
+    public ObservableBoolean selected = new ObservableBoolean();
```

Comment:
Please use Boolean. We are removing the use of Observable fields.

Reply:
Also, move this property outside the model if possible